### DISCUSSION TITLE
[Bug report] Issue with Simple Theme - Table of Content

### DISCUSSION THREAD

**Discussion by eudaimon-pol** [GetPublii/Publii#1711] [Bug Report] [answered]
### Operating system

Windows 10 Home (22H2)

### Publii version

Version: 0.46.1 (build 16918)

### Issue type

Free theme

### Bug description

After changing theme of my existing website to Simple theme (3.0) I have noticed that table of content is being formatted/ previewed wrongly. Instead of being separated from previous paragraph, it seems to be attached to a paragraph above it.

![Screenshot](https://github.com/user-attachments/assets/a223e2c7-9490-4883-b19c-2ccddb33ba2f)

**Comment by bobmitro** [comment, marked as answer]
Fixed in Simple v. 3.0.1.0 
https://marketplace.getpublii.com/themes/simple/
